p5-travel-app/src/client/js/weatherApi.js

```diff
@@ -0,0 +1,22 @@
+const weatherKey = "b3432a956d7b45a18ba374d67c837a2e";
+
+export async function getWeatherForecast(latitude, longitude) {
+  const endpoint =
+    "https://api.weatherbit.io/v2.0/forecast/daily?" +
+    `lat=${latitude}&lon=${longitude}&key=` +
+    weatherKey;
+  //console.log(endpoint);
```

Comment:
remove console

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)